pkg/sdn/plugin/controller.go

```diff
@@ -50,7 +51,8 @@ func getPluginVersion(multitenant bool) []string {
 func alreadySetUp(multitenant bool, localSubnetGatewayCIDR string) bool {
 	var found bool
 
-	itx := ipcmd.NewTransaction(LBR)
+	kexec := kexec.New()
```

Comment:
overwriting the package name with an instance of the object makes it really hard to follow code using this